### ISSUE TITLE
Using remotemoe causes too many arguments error and remotemoe link to change after every session

### ISSUE THREAD

**Issue by OfficialDSplayer** [Bing-su/sd-webui-tunnels#4]
<img width="468" alt="image" src="https://user-images.githubusercontent.com/66536577/233291953-8c5b4988-4919-48bf-997b-b0d3e49a716f.png">

Prior to uninstalling and reinstalling this extension, I did not have this error and was using the same remotemoe link for each session.

**Comment by Bing-su** [comment]
I've fixed the suspicious one. Please check it out.

**Comment by OfficialDSplayer** [comment]
> I've fixed the suspicious one. Please check it out.

Thanks that patch was able to fix the issue.

**Comment by OfficialDSplayer** [comment]
Btw it seems like I found what the initial issue was besides the arguments being a problem. I had moved my stable-diffusion-webui folder from the original folder that it is in (C:\Users\USERNAME\Stable Diffusion\stable-diffusion-webui) and to another drive (G:\Stable Diffusion\stable-diffusion-webui) and when I reinstalled the extension, I guess it didn't like it being in a different place. Pretty interesting but thanks again for fixing it.
